lsp-pyright.el

```diff
@@ -106,6 +106,30 @@ If there are no execution environments defined in the config file."
   :group 'lsp-pyright)
 (make-variable-buffer-local 'lsp-pyright-extra-paths)
 
+(defcustom lsp-pyright-auto-import-completions t
+  "Determines whether pyright offers auto-import completions."
+  :type 'boolean
+  :group 'lsp-pyright)
+
+(defcustom lsp-pyright-stub-path ""
+  "Path to directory containing custom type stub files."
+  :type 'string
+  :group 'lsp-pyright)
+
+(defcustom lsp-pyright-venv-path ""
```

Comment:
Suggest: ehe initial value is nil.

Reply:
We need to send `""` to the server, otherwise it will cause `TypeError` and fail to read the rest of the configuration. 
I set it to `nil` here and send `""` to the server if `venv` can't be found.